.ci/diff-report.sh

```diff
@@ -95,6 +95,11 @@ parse-pr-description-text)
   NEW_MODULE_CONFIG_LINK=$(echo "$NEW_MODULE_CONFIG_PARAMETER" | sed -E 's/New module config: //')
   PATCH_CONFIG_LINK=$(echo "$PATCH_CONFIG_PARAMETER" | sed -E 's/Diff Regression patch config: //')
   REPORT_LABEL=$(echo "$REPORT_LABEL_PARAMETER" | sed -E 's/Report label: //')
+  # trim
```

Comment:
The diff in your PR is of 925 commits and 3671 changed files. The branch head amust have got updated with merge commits. Please reset your he